geometry_msgs/Pose current_pose
geometry_msgs/Pose target_pose
---
bool success
string message
